web/models/pair.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Pairmotron.Pair do
+  use Pairmotron.Web, :model
+
+  schema "pairs" do
+    field :year, :integer
+    field :week, :integer
+    field :pair_group, :integer
+    belongs_to :user, Pairmotron.User
```

Comment:
Yep, that makes sense, especially for when we extend Pair for session information. Will do this.